google-cloud/README.md

```diff
@@ -1,14 +1,47 @@
 # google-cloud
 
-This gem is a convenience package for loading all gems in the google-cloud
-project. Users are encouraged to depending on the individual gems needed for
-your project instead of using this gem with every dependency. Please see the
-[GitHub repository](https://github.com/GoogleCloudPlatform/google-cloud-ruby)
-for more information about the individual google-cloud gems.
+The [google-cloud](https://github.com/GoogleCloudPlatform/google-cloud-ruby/tree/master/google-cloud)
+gem is a convenience package that lazily loads the vast majority of the
+[google-cloud-*](https://github.com/GoogleCloudPlatform/google-cloud-ruby) gems.
+Because there are now so many google-cloud-* gems, instead of using this gem in
+your production application, we encourage you to directly require only the
+individual google-cloud-* gems that you need.
 
 - [google-cloud API documentation](http://googlecloudplatform.github.io/google-cloud-ruby/docs/google-cloud/latest)
 - [google-cloud on RubyGems](https://rubygems.org/gems/google-cloud)
 
+## List of dependencies
+
+This gem depends on and lazily loads the following google-cloud-* gems:
+
+- [google-cloud-bigquery](https://googlecloudplatform.github.io/google-cloud-ruby/docs/google-cloud-bigquery)
```

Comment:
What do you think about linking to `latest`?

Reply:
Sure, probably a wise choice, will do.

Reply:
Actually, the way it is now might be good in the sense that users must explicitly choose the version... a little bit of hassle, but we have had issues in the past w/ people trying to use examples from the latest docs when they are using older versions.

Reply:
@dazuma Any input on this debate? 

Should the meta-package readme link directly to the docs for the latest version of a member gem ([example](http://googlecloudplatform.github.io/google-cloud-ruby/docs/google-cloud-logging/v1.5.4/)), versus linking to the list of versions for the gem ([example](http://googlecloudplatform.github.io/google-cloud-ruby/docs/google-cloud-logging/)). Stackdriver currently links to the list of versions, so if I change this PR to link to the latest, the Stackdriver readme should probably also change its links.

Reply:
Stackdriver also has instrumentation links, which used to be outside of any particular version, but with all of the other documentation changes those links are now resolving to the most recent gem.

This URL:

http://googlecloudplatform.github.io/google-cloud-ruby/docs/stackdriver/latest/

links to this URL:

https://googlecloudplatform.github.io/google-cloud-ruby/docs/google-cloud-debugger/latest/file.INSTRUMENTATION

Reply:
My leaning on this is to link to the version list, because if we link into a particular version (or latest), there isn't an obvious way to navigate to other versions, or even to tell that we are in fact looking at a version-specific piece of documentation.

For links to specific content (e.g. instrumentation.md), we kinda don't have a choice—there is no "version list" for individual files, so we have to link directly to a specific version of the file. But for general links like these in the gem list, I think it's fine to go to the version list and make it explicit to users that they have to pay attention to the version.

Reply:
kk, makes sense.